### ISSUE TITLE
How much the loss can be convergence？

### ISSUE THREAD

**Issue by cc50121** [fudan-zvg/PolarFormer#10]
I have trained the vovnet version for 18 epcochs, but the total loss is still about 12.

**Comment by yanqinJiang** [comment]
Sorry, I cannot find the log of polarformer-vovnet. But the log of[ polarformer-r101-w/o-bev_aug](https://drive.google.com/file/d/13hwLWauwTE9i2K2_-w8pNlfTKj9N1Jbl/view) indicates that the loss converges at around 3.8 after 24 epochs, and I remember polarformer-vovnet converges much faster and ends up with a lower value than polarformer-r101, maybe around 3 or below 3. (The vovnet backbone I use was pretrained on nuScenes trainval set #8 ,  so polarformer-vovnet is supposed to converge faster.)

Maybe you can share your training log here and I will check if there is anything wrong. : )
